src/Integration.Vsix.UnitTests/Suppression/DelegateInjectorTests.cs

```diff
@@ -52,9 +53,20 @@ class SonarAnalysisContext
 }";
         private const string SonarAnalyzerAssemblyName = "SonarAnalyzer";
 
+        private static IServiceProvider dummyServiceProvider = new Mock<IServiceProvider>().Object;
+        private static Func<Diagnostic, bool> dummyFunction = diag => true;
+
         [TestMethod]
         public void Injector_ValidPreLoadedAssemblies_SetsPropertyOk()
         {
+            using (var wrapper = new TestDomainWrapper<DelegateInjectorTests>())
+            {
+                wrapper.RemoteObject.Execute_ValidPreLoadedAssemblies_SetsPropertyOk();
```

Comment:
The tests now all following this pattern: create a new app domain, then call the _ExecuteXXX_ method that will be run in the remote domain. Exceptions are propagated between the app domains, so an assertion failure in the remote domain will cause a test failure as normal.

I should have updated the comment at the top of the test file - I'll do that.